src/main/java/org/example/springboot/repository/BookRepositoryImpl.java

```diff
@@ -0,0 +1,50 @@
+package org.example.springboot.repository;
+
+import java.util.List;
+import org.example.springboot.model.Book;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
use Lombok annotation to generate constructor
```suggestion
@Repository
@RequiredArgsConstructor
```